elasticsearch_dsl/utils.py

```diff
@@ -151,6 +158,7 @@ class AttrDict(Generic[_ValT]):
     """
 
     _d_: Dict[str, _ValT]
+    RESERVED: Dict[str, str] = {"from_": "from"}
```

Comment:
AttrDict now automatically recognizes `from` keys and exposes them as `from_`.

Reply:
How are you thinking that would work? It seemed to me the AttrDict needs to know that when I say `whateter.from_` I'm really saying `whatever["from"]`. This is going to happen in the application, not in generated code, so I have no control except from inside the AttrDict. If you can think of a better option let me know.